packages/compat/src/compat-app-builder.ts

```diff
@@ -1443,9 +1524,9 @@ function stringOrBufferEqual(a: string | Buffer, b: string | Buffer): boolean {
   return false;
 }
 
-const babelFilterTemplate = compile(`
-const { babelFilter } = require(${JSON.stringify(require.resolve('./index.js'))});
-module.exports = babelFilter({{json-stringify skipBabel}}, "{{js-string-escape appRoot}}");
+const babelFilterTemplate = jsHandlebarsCompile(`
+const { babelFilter } = require(${JSON.stringify(require.resolve('@embroider/core'))});
+module.exports = babelFilter({{json-stringify skipBabel}}}, "{{{js-string-escape appRoot}}");
```

Comment:
this does not look right, or am I missing something?

Reply:
I didn't get the merge conflict right on the first try.